### PR TITLE
{devel}[foss/2016b] Yade 2016.06a (REVIEW)

### PR DESCRIPTION
**Pull request by jrbosch**
bump to lasted release of Yade (2016.06a) with foss 2016b

### CODE REVIEW

File: easybuild/easyconfigs/q/Qt5/Qt5-5.7.0-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [
+    'http://download.qt.io/official_releases/qt/%(version_major_minor)s/%(version)s/single/',
+    'http://download.qt.io/archive/qt/%(version_major_minor)s/%(version)s/single/'
+]
+sources = ['qt-everywhere-opensource-src-%(version)s.tar.xz']
+
+patches = ['%(name)s-%(version)s-workaround-icpc-bug.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/q/Qt5/Qt5-5.7.0-foss-2016b.eb]: @jrbosch You can drop this, it's only required when building with the Intel compilers

Comment (boegel) [inline easybuild/easyconfigs/q/Qt5/Qt5-5.7.0-foss-2016b.eb]: @jrbosch You can drop this, it's only required when building with the Intel compilers

File: easybuild/easyconfigs/y/Yade/Yade-2016.06a-foss-2016b-Python-2.7.12.eb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    ('IPython', '5.1.0', versionsuffix),
+    ('libQGLViewer', '2.6.3'),
+    ('Loki', '0.1.7'),
+    ('matplotlib', '1.5.3', versionsuffix),
+    ('minieigen', '0.5.3', versionsuffix),
+    ('PyGTS', '0.3.1', versionsuffix),
+    ('Python', '2.7.12'),
+    ('Qt5', '5.7.0'),
+    ('SQLite', '3.13.0'),
+    ('VTK', '7.0.0', versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/y/Yade/Yade-2016.06a-foss-2016b-Python-2.7.12.eb]: @jrbosch Are you sure Yade is compatible with VTK7? The `intel/2016b` easyconfig uses VTK 6.3.0, I seem to recall there was a good reason to stick to VTK6...

Comment (jrbosch) [inline easybuild/easyconfigs/y/Yade/Yade-2016.06a-foss-2016b-Python-2.7.12.eb]: that's true https://bugs.launchpad.net/yade/+bug/1586275
thanks @boegel

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 22 out of 22 (22 easyconfigs in this PR)
node2585.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bb4a59479a35873caba5ca9572b1b0ee for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 28 out of 28 (22 easyconfigs in this PR)
node2094.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e5b176bc704d00ecb8b54808c921a4c5 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jrbosch!

**Pull request merged**